omod-1.9/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/controller/openmrs1_9/VisitAttributeTypeController1_9Test.java

```diff
@@ -1,161 +1,161 @@
-/**
- * This Source Code Form is subject to the terms of the Mozilla Public License,
- * v. 2.0. If a copy of the MPL was not distributed with this file, You can
- * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
- * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
- *
- * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
- * graphic logo is a trademark of OpenMRS Inc.
- */
-package org.openmrs.module.webservices.rest.web.v1_0.controller.openmrs1_9;
-
-import org.junit.Assert;
-import org.junit.Before;
-import org.junit.Test;
-import org.openmrs.VisitAttributeType;
-import org.openmrs.api.VisitService;
-import org.openmrs.api.context.Context;
-import org.openmrs.module.webservices.rest.SimpleObject;
-import org.openmrs.module.webservices.rest.test.Util;
-import org.openmrs.module.webservices.rest.web.RestConstants;
-import org.openmrs.module.webservices.rest.web.RestTestConstants1_9;
-import org.openmrs.module.webservices.rest.web.v1_0.controller.MainResourceControllerTest;
-
-import javax.servlet.http.HttpServletResponse;
-
-/**
- * Contains tests for {@link VisitAttributeTypeController} CRUD operations
- */
-public class VisitAttributeTypeController1_9Test extends MainResourceControllerTest {
```

Comment:
Did you intentionally delete all this red?

Reply:
Fixed that 😄